deploy/docker/bff_api_list.json

```diff
@@ -0,0 +1,4 @@
+[
+    {"name":"server", "url": "http://ogree-core_arango_api:8080"},
+    {"name":"objects", "url": "http://ogree-core_api:3001"}
```

Comment:
the containers could have a different name depending on the COMPOSE_PROJECT_NAME environment variable

Reply:
Yes, you need to change this config to match your deployement...

Reply:
The superadmin backend will need to do that then, so the deployment works directly without intervention of the user.